application/controllers/template.php

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>{pagetitle}</title>
+        <meta HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8"/>
+        <link rel="stylesheet" type="text/css" media="all" href="css/reset.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="css/text.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="css/style.css" />
+        <link rel="stylesheet" type="text/css" media="all" href="css/lightbox.css" />
+    </head>
+    <body>
+        <div id="wrapper">
+            <div id="header">
+                <span class="myhead">Lightning Air</span>
+                <span class="mynav">
+                    <ul>
+                        <li><a href="/">Home</a></li>
+                        <li><a href="/flights">Flights</a></li>
+                        <li><a href="/fleet">Fleet</a></li>
+                    </ul>
+                </span>
+            </div>
+            <div class="alone"></div>
+            <div id="content">
+              {content}
+            </div>
+            <div id="footer" class="span12">
+                Copyright &copy; 2017, Lightning Air Inc.</a>.
+            </div>
+        </div>
+        <script type="text/javascript" src="js/jquery-1.11.0.min.js"></script>
+        <script type="text/javascript" src="js/lightbox.min.js"></script>
+    </body>
+</html>
```

Comment:
I think I prefer the quotes template instead of the gallery one. Could you update the template to follow this [template ](https://github.com/jedi-academy/starter-route66/blob/master/application/views/template.php) instead?

Reply:
done